read-fonts/src/collections.rs

```diff
@@ -0,0 +1,4 @@
+//! data structures useful for font work
```

Comment:
```suggestion
//! Data structures useful for font work.
```
Hurts my eyes when public docs are missing capitalization and punctuation :)